src/components/charts/scatter-plot/scatter-plot-styles.module.scss

```diff
@@ -70,6 +70,7 @@ $chart-margin: 14px;
 .tickValue {
   @extend %annotation;
   color: $alto;
+  font-size: 18px;
```

Comment:
Could you use a variable for this? It's true that we dont have any variable for font-size: 18px. But maybe create a new one on `src/styles/settings.scss`